src/Meilisearch/PaginatedSearchResult.cs

```diff
@@ -9,15 +9,23 @@ namespace Meilisearch
     /// <typeparam name="T">Hit type.</typeparam>
     public class PaginatedSearchResult<T> : ISearchable<T>
     {
-        public PaginatedSearchResult(IReadOnlyCollection<T> hits, int hitsPerPage, int page, int total,
+        public PaginatedSearchResult(
```

Comment:
I reorganized the attributes here (one in each line). But I also added `int totalPages` and replaced `int total` with `int totalHits`.

I'm unsure how this can affect the user because it does not seem to be used (I think the json handler takes care of that).

Reply:
I'm not sure to understand, Do you mean the json handler should transform `int total` by `totalHits`?

Reply:
Yes, because the constructor does not seem to be used anywhere.